### PR TITLE
std.socket unittest doc

### PR DESCRIPTION
**Pull request by burner**
* One example could be moved to unittest doc

### CODE REVIEW

File: std/socket.d [hunk 1]
```diff
@@ -1678,6 +1673,13 @@ public:
             this.sin.sin_port == other.sin.sin_port;
     }
 
+    ///
+    @system unittest
+    {
+        InternetAddress addr1,addr2;
+        assert(addr1 == addr2);
+    }
```
Comment (quickfur) [inline std/socket.d]: What about a non-trivial example?

Comment (burner) [inline std/socket.d]: done

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**